Library/Formula/scw.rb

```diff
@@ -42,17 +26,17 @@ def install
     ln_s buildpath, buildpath/"src/github.com/scaleway/scaleway-cli"
     Language::Go.stage_deps resources, buildpath/"src"
 
-    cd "src/github.com/tools/godep" do
-      system "go", "install"
-    end
-
-    system "./bin/godep", "go", "build", "-o", "scw"
+    system "go", "build", "-o", "scw", "."
     bin.install "scw"
 
     bash_completion.install "contrib/completion/bash/scw"
     zsh_completion.install "contrib/completion/zsh/_scw"
   end
 
+  def caveats
+    "Use `scw login` to set up the correct environment to use scaleway-cli."
+  end
```

Comment:
Ok, I based this change from https://github.com/Homebrew/homebrew/blob/master/Library/Formula/s3sync.rb#L14-L16.
Reverting now